x-pack/plugins/observability_solution/synthetics/public/apps/synthetics/components/alerts/common/fields.tsx

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { FieldPopoverExpression } from './field_popover_expression';
+import { Suggestion } from '../hooks/use_fetch_synthetics_suggestions';
+import { FieldSelector } from './field_selector';
+
+interface FieldProps {
+  value?: string[];
+  onChange: (value?: string[]) => void;
+  setSearch: (val: string) => void;
+  suggestions?: Suggestion[];
+  allSuggestions?: Record<string, Suggestion[]>;
+  isLoading?: boolean;
+  setSelectedField: (value?: string) => void;
+  selectedField?: string;
+}
+
+export const allOptionText = i18n.translate('xpack.synthetics.filter.alert.allLabel', {
+  defaultMessage: 'All',
+});
+
+export function MonitorField({ value, onChange, ...rest }: FieldProps) {
```

Comment:
```suggestion
export function MonitorField({ value, onChange, ...props }: FieldProps) {
```

super nit: I know it kinda breaks with convention about spreading/accumulating fields in this way, but I find if you're looking at the code below without looking at the declaration, in the context of React components `props` is a way better name than `rest`, which usually requires the reader to check further up.